src/Answers.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import './App.css';
+
+const Answers = ({answers}) => {
+  let answersHTML = answers.map((answer, index) => {
+    return (
+      <p className="quiz-answer" key={index}>{answer.title}</p>
```

Comment:
thanks for the link!  I think this is what you are looking for.  I added this to the input: 

`name={'answer-group-'+index} `